templates/myaccount/downloads.php

```diff
@@ -37,7 +37,7 @@
 <?php else : ?>
 	<div class="woocommerce-Message woocommerce-Message--info woocommerce-info">
 		<a class="woocommerce-Button button" href="<?php echo esc_url( apply_filters( 'woocommerce_return_to_shop_redirect', wc_get_page_permalink( 'shop' ) ) ); ?>">
-			<?php esc_html_e( 'Go shop', 'woocommerce' ); ?>
+			<?php esc_html_e( 'Go to the shop', 'woocommerce' ); ?>
```

Comment:
I am wondering if we should not maybe change the wording here. "Go to the shop" does not sound right to me. Perhaps something along the lines of "Browse products" or "Browse shop" is better suited?

Reply:
I'm fine with both, but "Browse products" sounds good.

Reply:
@rodrigoprimo you marked it as "needs feedback" but do you have any suggestion for it?
I marked to "needs review" because I hope someone else of the team can help you make a choice of words here. Can you help?

Reply:
@claudiosanches I was going through the list of issues labeled as "needs review". When I started checking this one, I noticed that @kloon had already reviewed it and it was pending a resolution to his suggestion. That is why I changed the label from "needs review" to "needs feedback". It wasn't clear to me that you were looking for feedback from someone else from the team.

I agree with your suggestion of using the same sentence for both the link displayed in the downloads page and the one displayed in the orders page but, as a non-native English speaker, I'm not sure what is best. Sorry for not being able to help much here.